README.md

```diff
@@ -152,6 +152,36 @@ preview = true
 
 Run `fortitude explain` to see which rules are in preview mode.
 
+### Excluding Files
+
+> [!NOTE]
+> To be featured in v0.7.0
+
+Fortitude will automatically ignore files in some directories (`build/`, `.git/`,
+`.venv/`, etc.), and this behaviour can be extended using the `--exclude` option. For
+example, to ignore all files in the directory `benchmarks/`:
```

Comment:
Oh, does `--exclude` not override the builtins?

Reply:
From playing around with Ruff, apparently not, as it behaves as if the built-ins are a mandatory set of `--extend-exclude` instead of `--exclude`. If you want it to lint everything in your `venv`, you have to manually pass it `venv/lib/Python3.10/site-packages`, and setting your own `exclude` doesn't change this behaviour.

I haven't played around with `include` in a config file, so I'm not sure how that might affect it.